gateway/1.9/modules/ROOT/pages/flex-conn-replica-caching.adoc

```diff
@@ -0,0 +1,26 @@
+= Configuring Replica Configuration Caching in Connected Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Enable replica configuration caching to have a Flex Gateway replica cache its gateway configuration in Redis shared storage. When configured, new replicas use the cached gateway configuration instead of downloading the configuration from Anypoint Platform. Caching a replica's configuration reduces scale-up time and increases resiliency.
+
+Replica caching works by:
+
+. When the first Flex Replica is created, it sees that their is no configuration in the Redis shared storage.
+. The first replica creates the group ID and downloads its configuration from Anypoint Platform. The replica then stores the configuration in Redis.
+. When a new Flex Replica is created, it uses the stored configuration.
+. At their start up, all Flex Replicas connect to Anypoint Platform and send configuration update requests to the same group ID.
+. When the configuration is updated on Anypoint Platform, only one replica receives the configuration updates.
```

Comment:
"only one replica receives the configuration updates"
This is an internal implementation detail that is not needed here

Reply:
we can just say that updates are downloaded and stored in redis for all replicas to use